sum.c

```diff
@@ -0,0 +1,40 @@
+/* Ashank Kumar, user inputs an array, proram outputs cumulative sum of the elements in the array so far */
+#include <stdio.h>
+
+int main()
+{
+	int x;
+	int total = 0;
+
+	printf("Enter the length of the string:");
```

Comment:
Perfect job! The only thing is I think you meant to write "length of the array" but that's not a big deal at all